CONTRIBUTORS.txt

```diff
@@ -25,24 +25,29 @@ needs to be listed here.
 - Ben Cohen <ben_cohen@apple.com>
 - Bouke Haarsma <bouke@haarsma.eu>
 - Chris Eidhof <chris@eidhof.nl>
+- Christian Weinberger <christian.weinberger@me.com>
 - Cory Benfield <cbenfield@apple.com>
 - Cotton Hou <himcotton@gmail.com>
 - Damiaan Dufaux <damiaan@dpro.be>
 - Daniel Alm <Daniel.Alm@ForumD.net>
 - Daniel Dunbar <daniel_dunbar@apple.com>
 - Dario Rexin <drexin@apple.com>
+- David Evans <d.evans@apple.com>
 - David Skrundz <david.skrundz@gmail.com>
+- Emil Marashliev <emarashliev@users.noreply.github.com>
 - Esteban Torres <me@estebantorr.es>
 - Eugen <eugenf78@gmail.com>
 - Frank Kair <frankkair@gmail.com>
 - Franz Busch <privat@franz-busch.de>
 - George Barnett <gbarnett@apple.com>
 - Gopal Sharma <gopal@the-sharmas.org>
-- Gwynne Raskind <graskind@apple.com>
+- Gwynne Raskind <gwynne@darkrainfall.org>
+- Heidi Puk Hermann <32973281+heidipuk@users.noreply.github.com>
 - Helge Heß <helge@alwaysrightinstitute.com>
 - Ian Partridge <i.partridge@uk.ibm.com>
 - Ilya Puchka <ilyapuchka@gmail.com>
 - JP Simard <jp@jpsim.com>
+- Jake <2bjake@users.noreply.github.com>
```

Comment:
@2bjake  if you tell me which email address to use, I can use a real email address instead of this odd github one.

Reply:
(and I can also add a real name)

Reply:
Jake Foster <jakef@fastmail.com>

Reply:
thanks, done